app/locale/en.coffee

```diff
@@ -963,7 +964,7 @@
     average_levels: "Average levels completed:"
     total_levels: "Total levels completed:"
     furthest_level: "Furthest level completed:"
-    concepts_covered: "Concepts Covered"
+    # concepts_covered: "Concepts Covered"
```

Comment:
How we deal with rescoping things that have already been translated? Like, some got moved from `courses` to `teachers`, including that one.